starter-code/scripts/article.js

```diff
@@ -2,9 +2,16 @@
 
 var articles = [];
 
-function Article (rawDataObj) {
+function Article (rawData) {
```

Comment:
You named your input here rawData, which matches the name of our data saved in blogArticles.js, which is not what we want. We do not want to name a parameter the same thing as a variable. The starter code gave you rawDataObj and you needed to `Save ALL the properties of `rawDataObj` into `this``.